plugins/modules/redfish_powerstate.py

```diff
@@ -118,26 +148,77 @@
 import json
 import re
 from ssl import SSLError
+from ansible.module_utils.compat.version import LooseVersion
 from ansible_collections.dellemc.openmanage.plugins.module_utils.redfish import Redfish, RedfishAnsibleModule
 from ansible.module_utils.six.moves.urllib.error import URLError, HTTPError
 from ansible.module_utils.urls import ConnectionError
 
+
+CHANGES_FOUND = "Changes found to be applied."
+NO_CHANGES_FOUND = "No Changes found to be applied."
+FIRM_VER_URI = "/redfish/v1/Managers/iDRAC.Embedded.1?$select=FirmwareVersion"
+INVALID_RESET_TYPE_OEM = "'{option}' is not supported. The supported values" \
```

Comment:
'{option}'

remove this single quote, keep it in simple sentence.

Reply:
This is for better understanding of the user.